DQM/SiPixelPhase1Track/plugins/SiPixelPhase1TrackClusters.cc

```diff
@@ -99,10 +110,29 @@ namespace {
     pixelClusterShapeCacheToken_ =
         consumes<SiPixelClusterShapeCache>(iConfig.getParameter<edm::InputTag>("clusterShapeCache"));
 
-    trackerTopoToken_ = esConsumes<TrackerTopology, TrackerTopologyRcd>();
-    trackerGeomToken_ = esConsumes<TrackerGeometry, TrackerDigiGeometryRecord>();
+    trackerTopoToken_ = esConsumes<TrackerTopology, TrackerTopologyRcd, edm::Transition::BeginRun>();
+    trackerGeomToken_ = esConsumes<TrackerGeometry, TrackerDigiGeometryRecord, edm::Transition::BeginRun>();
     clusterShapeHitFilterToken_ =
         esConsumes<ClusterShapeHitFilter, CkfComponentsRecord>(edm::ESInputTag("", "ClusterShapeHitFilter"));
+    templateDBobjectToken_ =
+        esConsumes<SiPixelTemplateDBObject, SiPixelTemplateDBObjectESProducerRcd, edm::Transition::BeginRun>();
```

Comment:
```suggestion
        esConsumes<edm::Transition::BeginRun>();
```
this could be simplified.